README.md

```diff
@@ -1 +1,147 @@
-# Leaderboard-M2W4
\ No newline at end of file
+# Leaderboard-M2W4
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>README</b></h3>
+
+</div>
```

Comment:
- [ **OPTIONAL** ] Kindly update the logo because it is broken. I will mark it as optional because it is a small change and I don't want to turn you because of this, but please update it because your project will look more professional
![image](https://user-images.githubusercontent.com/62705947/208942892-9514a3d8-3dad-48a7-af99-defd71cc00a7.png)

Reply:
@yash244466666, you were quicker than me for a few seconds :laughing: